src/website/app/demos/Pagination/examples/kitchenSink.js

```diff
@@ -0,0 +1,80 @@
+/* @flow */
+import { Component } from 'react';
+import Pagination from '../../../../../library/Pagination';
+import Table from '../../../../../library/Table';
+import DemoLayout from '../../shared/DemoLayout';
+import { columns, data } from '../shared/data';
+
+export default {
+  id: 'kitchen-sink',
+  title: 'Kitchen Sink',
+  hideFromProd: true,
+  scope: {
+    Component,
+    columns,
+    data,
+    DemoLayout,
+    Pagination,
+    Table
+  },
+  source: `
+    () => {
+      const pageSizes = [2, 3, 4];
+      const category = 'minerals';
```

Comment:
Again, having these way up here seems to make the example feel more complex.  I'd suggest to simplify by inlining the values in the JSX and just hardcoding a pageSize for the initial state.